modules/nw-ovn-kubernetes-change-log-levels.adoc

```diff
@@ -86,16 +86,41 @@ configmap/env-overrides.yaml created
 [source,terminal]
 ----
 $ oc delete pod -n openshift-ovn-kubernetes \
---field-selector spec.nodeName=ip-10-0-217-114.ec2.internal -l app=ovnkube-node
+--field-selector spec.nodeName=ci-ln-3njdr9b-72292-5nwkp-master-0 -l app=ovnkube-node
 ----
 +
 [source,terminal]
 ----
 $ oc delete pod -n openshift-ovn-kubernetes \
---field-selector spec.nodeName=ip-10-0-209-180.ec2.internal -l app=ovnkube-node
+--field-selector spec.nodeName=ci-ln-3njdr9b-72292-5nwkp-master-2 -l app=ovnkube-node
 ----
 +
 [source,terminal]
 ----
-$ oc delete pod -n openshift-ovn-kubernetes -l app=ovnkube-master
+$ oc delete pod -n openshift-ovn-kubernetes -l app=ovnkube-node
+----
+
+. To verify all nodes for a specific pod, run the following command:
++
+[source,terminal]
+----
+$ oc logs -n openshift-ovn-kubernetes --all-containers --prefix ovnkube-node-<xxxx> | grep -E -m 10 '(Logging config:|vconsole|DBG)'
```

Comment:
From https://issues.redhat.com/browse/OCPBUGS-18412
It looks like I also suggested

```shell
for f in $(oc -n openshift-ovn-kubernetes get po -l 'app=ovnkube-control-plane' --no-headers -o custom-columns=N:.metadata.name) ; do echo "---- $f ----" ; oc -n openshift-ovn-kubernetes exec -c ovnkube-cluster-manager $f --  pgrep -a ovnkube ; done
```